### DISCUSSION TITLE
KEDA v2.12.1

### DISCUSSION THREAD

**Discussion by zroubalik** [kedacore/keda#5216] [Announcements]
[KEDA 2.12.1](https://github.com/kedacore/keda/releases/tag/v2.12.1)

We are happy to release 2.12.1 🎉

Learn how to deploy KEDA by reading [our documentation](https://keda.sh/docs/2.12/deploy/).

🗓️ The next KEDA release is currently being estimated for January 11th, 2024 _(planned later due to holidays)_, learn more in our [roadmap](https://github.com/kedacore/keda/blob/main/ROADMAP.md#upcoming-release-cycles).

### Fixes

- **General**: Fix CVE-2023-39325 in golang.org/x/net ([#5122](https://github.com/kedacore/keda/issues/5122))
- **General**: Fix CVE-2023-45142 in Opentelemetry ([#5089](https://github.com/kedacore/keda/issues/5089))
- **General**: Fix logger in Opentelemetry collector ([#5094](https://github.com/kedacore/keda/issues/5094))
- **General**: Fix otelgrpc DoS vulnerability ([#5208](https://github.com/kedacore/keda/issues/5208))
- **General**: Prevented stuck status due to timeouts during scalers generation ([#5083](https://github.com/kedacore/keda/issues/5083))
- **Azure Pipelines**: No more HTTP 400 errors produced by poolName with spaces ([#5107](https://github.com/kedacore/keda/issues/5107))

## What's Changed
**Full Changelog**: https://github.com/kedacore/keda/compare/v2.12.0...v2.12.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/keda/releases/tag/v2.12.1'>KEDA v2.12.1</a>.</em>

**Comment by Hoyusalon** [comment]
> [KEDA 2.12.1](https://github.com/kedacore/keda/releases/tag/v2.12.1)
> 
> 
> 
> We are happy to release 2.12.1 🎉
> 
> 
> 
> 
> 
> Learn how to deploy KEDA by reading [our documentation](https://keda.sh/docs/2.12/deploy/).
> 
> 
> 
> 🗓️ The next KEDA release is currently being estimated for January 11th, 2024 _(planned later due to holidays)_, learn more in our [roadmap](https://github.com/kedacore/keda/blob/main/ROADMAP.md#upcoming-release-cycles).
> 
> 
> 
> ### Fixes
> 
> 
> 
> - **General**: Fix CVE-2023-39325 in golang.org/x/net ([#5122](https://github.com/kedacore/keda/issues/5122))
> 
> - **General**: Fix CVE-2023-45142 in Opentelemetry ([#5089](https://github.com/kedacore/keda/issues/5089))
> 
> - **General**: Fix logger in Opentelemetry collector ([#5094](https://github.com/kedacore/keda/issues/5094))
> 
> - **General**: Fix otelgrpc DoS vulnerability ([#5208](https://github.com/kedacore/keda/issues/5208))
> 
> - **General**: Prevented stuck status due to timeouts during scalers generation ([#5083](https://github.com/kedacore/keda/issues/5083))
> 
> - **Azure Pipelines**: No more HTTP 400 errors produced by poolName with spaces ([#5107](https://github.com/kedacore/keda/issues/5107))
> 
> 
> 
> ## What's Changed
> 
> **Full Changelog**: https://github.com/kedacore/keda/compare/v2.12.0...v2.12.1
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/keda/releases/tag/v2.12.1'>KEDA v2.12.1</a>.</em>

**Reply by tomkerkhove** [reply]
Hi, did you have a question?
